<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialconsole</class>
 <widget class="QMainWindow" name="serialconsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Port Console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="console">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>771</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="serialPortLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>SerialPort</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>691</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendPushButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectedLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>81</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Unconnected</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
